README.md

```diff
@@ -1,12 +1,23 @@
 # Microsoft Authentication Library for JavaScript (MSAL.js)
 
-The Microsoft Authentication Library for JavaScript enables client-side JavaScript web applications, running in a web browser, to authenticate users using [Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview) for work and school accounts (AAD), Microsoft personal accounts (MSA), and social identity providers like Facebook, Google, LinkedIn, Microsoft accounts, etc. through [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. It also enables your app to get tokens to access [Microsoft Cloud](https://www.microsoft.com/enterprise) services such as [Microsoft Graph](https://graph.microsoft.io).
+The Microsoft Authentication Library for JavaScript adds Auth to Javascript applications, and access to resource APIs in the [Microsoft Cloud](https://www.microsoft.com/enterprise) such as [Microsoft Graph](https://graph.microsoft.io).
 
-## Repository
+MSAL.js now supports Node.js and Electron! 
```

Comment:
I think references to Node/Electron should go in the MSAL Node readme, and should make it clear what Electron support means (there are different scenarios and I don't believe all of them are supported yet).

Reply:
+1